grammars/restructuredtext.cson

```diff
@@ -477,87 +477,11 @@ repository:
-        name: 'constant.character.math.tex'
-      }
-      {
-        captures:
-          '1':
-            name: 'punctuation.definition.constant.math.tex'
-        match: '(\\\\)(sum|prod|coprod|int|oint|bigcap|bigcup|bigsqcup|bigvee|bigwedge|bigodot|bigotimes|bogoplus|biguplus)\\b'
-        name: 'constant.character.math.tex'
-      }
-      {
-        captures:
-          '1':
-            name: 'punctuation.definition.constant.math.tex'
-        match: '(\\\\)(arccos|arcsin|arctan|arg|cos|cosh|cot|coth|csc|deg|det|dim|exp|gcd|hom|inf|ker|lg|lim|liminf|limsup|ln|log|max|min|pr|sec|sin|sinh|sup|tan|tanh)\\b'
-        name: 'constant.other.math.tex'
-      }
-      {
-        begin: '((\\\\)Sexpr(\\{))'
-        beginCaptures:
-          '1':
-            name: 'support.function.sexpr.math.tex'
-          '2':
-            name: 'punctuation.definition.function.math.tex'
-          '3':
-            name: 'punctuation.section.embedded.begin.math.tex'
-        contentName: 'support.function.sexpr.math.tex'
-        end: '(((\\})))'
-        endCaptures:
-          '1':
-            name: 'support.function.sexpr.math.tex'
-          '2':
-            name: 'punctuation.section.embedded.end.math.tex'
-          '3':
-            name: 'source.r'
-        name: 'meta.embedded.line.r'
-        patterns: [
-          {
-            begin: '\\G(?!\\})'
-            end: '(?=\\})'
-            name: 'source.r'
-            patterns: [
-              {
-                include: 'source.r'
-              }
-            ]
-          }
-        ]
-      }
-      {
-        captures:
-          '1':
-            name: 'punctuation.definition.constant.math.tex'
-        match: '(\\\\)([^a-zA-Z]|[A-Za-z]+)(?=\\b|\\}|\\]|\\^|\\_)'
-        name: 'constant.other.general.math.tex'
-      }
-      {
-        match: '(([0-9]*[\\.][0-9]+)|[0-9]+)'
-        name: 'constant.numeric.math.tex'
-      }
-      {
-        match: '«press a-z and space for greek letter»[a-zA-Z]*'
-        name: 'meta.placeholder.greek.math.tex'
-      }
-      {
-        match: '(?<!\\\\)[{}]'
-        name: 'punctuation.definition.arguments'
-      }
-  ]
-
 	# Intepreted text: Multiline with role
 	"math-inline":
 		name: "markup.other.command.restructuredtext"
 		contentName: "markup.math.inline.restructuredtext"
-		patterns: [{include: '#math-syntax'}]
+		patterns: [include: "text.tex.latex"]
```

Comment:
Try replacing `text.tex.latex` with `text.tex#math`, mate. =) That'll load the [named block](https://github.com/area/language-latex/blob/master/grammars/tex.cson#L148) from the grammar with the `text.tex` scope.